firefox_ui_harness/runners/base.py

```diff
@@ -11,64 +11,14 @@
 from firefox_puppeteer.testcases import FirefoxTestCase
 
 
-DEFAULT_PREFS = {
-    'app.update.auto': False,
-    'app.update.enabled': False,
-    'browser.dom.window.dump.enabled': True,
-    # Bug 1145668 - Has to be reverted to about:blank once Marionette
-    # can correctly handle error pages
-    'browser.newtab.url': 'about:newtab',
-    'browser.newtabpage.enabled': False,
-    'browser.reader.detectedFirstArticle': True,
-    'browser.safebrowsing.enabled': False,
-    'browser.safebrowsing.malware.enabled': False,
-    'browser.search.update': False,
-    'browser.sessionstore.resume_from_crash': False,
-    'browser.shell.checkDefaultBrowser': False,
-    'browser.startup.page': 0,
-    'browser.tabs.animate': False,
-    'browser.tabs.warnOnClose': False,
-    'browser.tabs.warnOnOpen': False,
-    'browser.uitour.enabled': False,
-    'browser.warnOnQuit': False,
-    'dom.ipc.reportProcessHangs': False,
-    'dom.report_all_js_exceptions': True,
-    'extensions.enabledScopes': 5,
-    'extensions.autoDisableScopes': 10,
-    'extensions.getAddons.cache.enabled': False,
-    'extensions.installDistroAddons': False,
-    'extensions.logging.enabled': True,
-    'extensions.showMismatchUI': False,
-    'extensions.update.enabled': False,
-    'extensions.update.notifyUser': False,
-    'focusmanager.testmode': True,
-    'geo.provider.testing': True,
-    'javascript.options.showInConsole': True,
-    'marionette.logging': False,
-    'security.notification_enable_delay': 0,
-    'signon.rememberSignons': False,
-    'startup.homepage_welcome_url': 'about:blank',
-    'toolkit.startup.max_resumed_crashes': -1,
-    'toolkit.telemetry.enabled': False,
-}
-
-
 class FirefoxUITestRunner(BaseMarionetteTestRunner):
     def __init__(self, **kwargs):
         BaseMarionetteTestRunner.__init__(self, **kwargs)
-
+        # pick up prefs from marionette_driver.geckoinstance.DesktopInstance
+        self.app = 'fxdesktop'
```

Comment:
What do you mean with `prefs` here? If that should be related to the moved preferences I don't think we have to mention that here. Better we say that we setup the right GeckoInstance by setting the appropriate `app`.